test/images/cuda-vector-add/README.md

```diff
@@ -15,4 +15,4 @@ $ make push
 ## Version history:
 
 1.0: build cuda-vector-add from CUDA 8.0.
-2.0: build amd64 cuda-vector-add from CUDA 10.0 and ppc64le from CUDA 9.2.
+2.0: build cuda-vector-add from CUDA 10.0
```

Comment:
should this be a 2.1?

Reply:
I think we can stay with 2.0. This change is to address the comment https://github.com/kubernetes/kubernetes/pull/73823#discussion_r254953524 that unfortunately wasn't caught before the submit.